test/addons/rook-cephfs/test

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+
+from drenv import kubectl
+
+PVC_NAME = "cephfs-pvc"
+SNAP_NAME = "cephfs-snap"
+
+
+def test_provisioning(cluster):
+    print(f"Deploying pvc {PVC_NAME} in cluster '{cluster}'")
+    kubectl.apply(
+        f"--filename={PVC_NAME}.yaml",
+        "--namespace=rook-ceph",
+        context=cluster,
+    )
+
+    print(f"Waiting until pvc {PVC_NAME} is bound in cluster '{cluster}'")
+    kubectl.wait(
+        f"pvc/{PVC_NAME}",
+        "--for=jsonpath={.status.phase}=Bound",
+        "--namespace=rook-ceph",
+        "--timeout=300s",
+        context=cluster,
+    )
+
+    print(f"Deploying snapshot {SNAP_NAME} in cluster '{cluster}'")
+    kubectl.apply(
+        f"--filename={SNAP_NAME}.yaml",
+        "--namespace=rook-ceph",
+        context=cluster,
+    )
+
+    print(f"Waiting until snapshot {SNAP_NAME} is bound in cluster '{cluster}'")
+    kubectl.wait(
+        f"volumesnapshot/{SNAP_NAME}",
+        "--for=jsonpath={.status.readyToUse}=true",
+        "--namespace=rook-ceph",
+        "--timeout=300s",
+        context=cluster,
+    )
```

Comment:
Cool to test snapshot, but how do we know that it is actually working? Could be useful to make change so the PVC, verify that they are available in the snapshot, and that new changes in the PVC are not.

Reply:
We are not testing CephFS snapshots or PVC we are testing if what we installed is functional. This request seems to be more than what is needed to setup the environment.

Reply:
This is what we do for the volsync test, it will help to debug issues when we know that the cephfs test works if ramen does not, and fail earlier before we try to install ramen if cephfs pvc or snapshot do not work.